src/fw/drivers/pmic/npm1300.c

```diff
@@ -176,16 +183,21 @@ bool pmic_init(void) {
 }
 
 bool pmic_power_off(void) {
-  /* TODO(FIRM-114): shut almost everything off, but not the nRF Vdd, and then put the nRF in system_off */
-  return false;
+  if (pmic_is_usb_connected()) {
+    PBL_LOG(LOG_LEVEL_ERROR, "USB is connected, cannot power off");
+    return false;
+  }
```

Comment:
also I wonder if this should trigger a reset, since this is often the user's intent when powering the device off

Reply:
I think this is actually avoided, at least in some paths (like PRF standby)